src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -44,23 +44,24 @@ class AppDeployIntegrationTest
     deploymentIds.length should be(1)
     val deploymentId = deploymentIds.head
 
-    val apiPostEvent = waitForEvent("api_post_event")
+    log.info("waiting for deployment success")
+    val events: Map[String, Seq[CallbackEvent]] = waitForEvents(
+      "api_post_event", "group_change_success", "deployment_info",
+      "status_update_event", "status_update_event",
+      "deployment_success")(30.seconds)
```

Comment:
In most other tests, the timeout is handled as optional parameter to enhance readability. We should do that here as well.